src/resolvers/projectResolver.test.ts

```diff
@@ -660,6 +660,93 @@ function projectsTestCases() {
       result.data.data.projects.projects[0].totalTraceDonations >= 100,
     );
   });
+  it('should return just listed projects, sort by acceptGiv, DESC', async () => {
+    await saveProjectDirectlyToDb({
+      ...createProjectData(),
+      title: String(new Date().getTime()),
+      totalTraceDonations: 100,
+      listed: false,
+    });
+    await saveProjectDirectlyToDb({
+      ...createProjectData(),
+      title: String(new Date().getTime()),
+      totalTraceDonations: 100,
+      listed: true,
+    });
+    const result = await axios.post(graphqlUrl, {
+      query: fetchAllProjectsQuery,
+      variables: {
+        limit: 50,
+        orderBy: {
+          field: 'AcceptGiv',
+          direction: 'DESC',
+        },
+      },
+    });
+    result.data.data.projects.projects.forEach(project => {
+      assert.isTrue(project.listed);
+    });
+  });
+  it('should return just listed projects, sort by Verified, DESC', async () => {
+    await saveProjectDirectlyToDb({
+      ...createProjectData(),
+      title: String(new Date().getTime()),
+      totalTraceDonations: 100,
+      verified: true,
+      listed: false,
+    });
+    await saveProjectDirectlyToDb({
+      ...createProjectData(),
+      title: String(new Date().getTime()),
+      totalTraceDonations: 100,
+      verified: true,
+      listed: true,
+    });
+    const result = await axios.post(graphqlUrl, {
+      query: fetchAllProjectsQuery,
+      variables: {
+        limit: 50,
+        orderBy: {
+          field: 'Verified',
+          direction: 'DESC',
+        },
+      },
+    });
+    result.data.data.projects.projects.forEach(project => {
+      assert.isTrue(project.listed);
+    });
+  });
+  it('should return just listed projects, sort by Traceable, DESC', async () => {
+    await saveProjectDirectlyToDb({
+      ...createProjectData(),
+      title: String(new Date().getTime()),
+      totalTraceDonations: 100,
+      verified: true,
+      traceCampaignId: 'campaignIdInTrace',
+      listed: false,
+    });
+    await saveProjectDirectlyToDb({
+      ...createProjectData(),
+      title: String(new Date().getTime()),
+      totalTraceDonations: 100,
+      traceCampaignId: 'campaignIdInTrace',
+      verified: true,
+      listed: true,
+    });
+    const result = await axios.post(graphqlUrl, {
+      query: fetchAllProjectsQuery,
+      variables: {
+        limit: 50,
+        orderBy: {
+          field: 'Traceable',
+          direction: 'DESC',
+        },
+      },
+    });
+    result.data.data.projects.projects.forEach(project => {
+      assert.isTrue(project.listed);
+    });
+  });
```

Comment:
These test cases were failing before changing projectResolver, but after that they all passed